pkg/kapis/devops/v1alpha3/register.go

```diff
@@ -55,10 +56,11 @@ func AddToContainer(container *restful.Container, devopsClient devopsClient.Inte
 		ws.Route(ws.GET("/devops/{devops}/credentials").
 			To(handler.ListCredential).
 			Param(ws.PathParameter("devops", "devops name")).
-			Param(ws.QueryParameter(params.PagingParam, "paging query, e.g. limit=100,page=1").
-				Required(false).
-				DataFormat("limit=%d,page=%d").
-				DefaultValue("limit=10,page=1")).
+			Param(ws.QueryParameter(query.ParameterName, "name used to do filtering").Required(false)).
+			Param(ws.QueryParameter(query.ParameterPage, "page").Required(false).DataFormat("page=%d").DefaultValue("page=1")).
+			Param(ws.QueryParameter(query.ParameterLimit, "limit").Required(false)).
+			Param(ws.QueryParameter(query.ParameterAscending, "sort parameters, e.g. reverse=true").Required(false).DefaultValue("ascending=false")).
```

Comment:
description misleading, should be `ascending=false`

Reply:
fixed